.woodpecker.yml

```diff
@@ -23,21 +59,26 @@ steps:
       # store cargo data in repo folder so that it gets cached between steps
       CARGO_HOME: .cargo
     commands:
+      - rustup component add rustfmt
```

Comment:
We do `cargo +nightly fmt` on all lemmy repos, so you do need to use the `rust:nightly` image for this.

Reply:
ok i will switch back to the nightly image